applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_geo_steady_state_piping_element.cpp

```diff
@@ -381,4 +381,124 @@ KRATOS_TEST_CASE_IN_SUITE(GeoSteadyStatePwPipingElementReturnsEquilibriumHeightF
     KRATOS_EXPECT_NEAR(pipe_height, 7.0, 1e-10);
 }
 
+KRATOS_TEST_CASE_IN_SUITE(GeoSteadyStatePwPipingElementReturnsPipeActive, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Arrange
+    const auto dummy_process_info = ProcessInfo{};
+    const auto p_properties       = std::make_shared<Properties>();
+
+    Model model;
+    auto& r_model_part = CreateModelPartWithWaterPressureVariableAndVolumeAcceleration(model);
+    auto  p_element =
+        CreateHorizontalUnitLengthGeoSteadyStatePwPipingElementWithPWDofs(r_model_part, p_properties);
+
+    p_element->SetValue(PIPE_ACTIVE, false);
+
+    std::vector<bool> pipe_active(
+        p_element->GetGeometry().IntegrationPointsNumber(p_element->GetIntegrationMethod()));
```

Comment:
I would supply an empty vector the `CalculateOnIntegrationPoints` and then check whether the size is correct after calling it.
```suggestion
    std::vector<bool> pipe_active_states;
```

Reply:
Indeed, the CalculateOnIntegrationsPoints resizes the result itself.